backend/functions/src/course/routes.ts

```diff
@@ -35,4 +42,32 @@ router.get('/students/:courseId', (req, res) => {
     })
 })
 
+router.get('/semester/current', (_, res) => {
+  getCurrentSemester()
+    .then((data) => res.status(200).send(data))
+    .catch((err) => {
+      logger.error(`Unexpected error getting semester: ${err.message}`)
```

Comment:
I think one thing you could do to make it a little refactored/useful is set up error logging like so: 

```javascript
.catch( (err) => { 
	const err_msg = `Unexpected error getting semester: ${err.msg}` 
	logger.error(err_msg); 
	res.status(500).send({ message: err_msg })
})	

```

Can also change the error message for the other endpoints to be like: 
```javascript 
`Error changing current semester to ${semester}`
```

So that the error message tells us more of what the user input is.